Documentation/Signoffs/Voice Activity Detection Subsystem.md

```diff
@@ -0,0 +1,108 @@
+# Voice Activity Detection Subsystem #
+
+## Subsystem Function:
+
+The purpose of this subsystem is to detect an individual's responsiveness or cognition through the use of call and response techniques. The subsystem will activate once an individual heartbeat has been detected. The speaker will play a sound with an instruction to say something. Once the instruction is given, the microphone will begin transmitting audio to a computing system which will then run an algorithm to detect if a human voice responded or not allowing the total system to know if the individual being tested has cognition or not.
+
+
+## Specs and Constraints			
+
+
+| Description | Constraint | Source of Constraint |
+|-------------|------------|----------------------|
+| (1) VAD System | Must detect the presence of a human voice from auddio signal | Functionality of subsytem with respect to triage algorithm |
+| (2) Microphone | Must pick up a minimum sound pressure level of 30 dB SPL from at least 1 meter away | Volume level of a human whisper. [2] DARPA distance constraint |
+| (3) Microphone | Must sense frequencies from 100 Hz to 3000 Hz | Frequency range of average man is 100-900 Hz while female is 300 - 3000 Hz.[1] |
+| (4) Speaker | Must output sound at a volume heard from at least 1 meter away | Individual can hear sound from the speaker |
+| (5) Speaker | Must play sound clips between 20 Hz- 20000 Hz | Frequency range an individual’s ear can hear [1] |
+| (6) Distance | Must function with noise from drone from at least 1 meter away | From DARPA constraints |
+| (7) Weight | he drone specified in conceptual design had a max load of 2.7 kg. Seeing as 4 subsystems will be attached to the drone. The max weight must not exceed 675g. | From DARPA constraints/ Conceptual design |
+| (8) Safety | Speaker output must not be louder than 120 dB | Levels at 120 dB can cause immediate harm to an individual's hearing [2] |  
+
+
+## Wiring Schematics
+
+
+
+![image](https://user-images.githubusercontent.com/79685126/235611053-69878c8b-1b4f-4cb6-9a22-76f6e17996e6.png)
+
+
+Figure1. Wire Diagram for the total subsystem
+
+## Analysis:
+
+(1)
```

Comment:
I'm not convinced this will detect human speech. it seems to just look to see something in the human frequency. which could be anything.